docker/shadow-hand/kinetic/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM shadowrobot/build-tools:xenial-kinetic-ide
+
+ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/master/ansible/deploy.sh"
+
+COPY additional_bashrc_cmds /tmp/additional_bashrc_cmds
+
+ENV PROJECTS_WS=/home/user/projects/shadow_robot
+RUN echo "Running one-liner" && \
+    apt-get update && \
+    wget -O /tmp/oneliner "$( echo "$remote_shell_script" | sed 's/#/%23/g' )" && \
+    chmod 755 /tmp/oneliner && \
+    gosu $MY_USERNAME /tmp/oneliner -w $PROJECTS_WS/shadow_robot/base -b "kinetic-devel" -v "kinetic"  && \
+    cat /tmp/additional_bashrc_cmds >> /home/user/.bashrc && \
+
+    echo "Removing cache" && \
```

Comment:
why are you cleaning cache and using apt-get update again. It is better to do it once at the end of all commands :)